docs/hopsml/hopsML.rst

```diff
@@ -109,42 +111,56 @@ Hops uses PySpark to manage resource allocation of CPU, Memory and GPUs. PySpark
     :scale: 35 %
     :figclass: align-center
 
-Hops supports cluster-wide Conda for managing Python library dependencies. Hops supports the creation of projects, and each project has its own conda environment, replicated at all hosts in the cluster. When you launch a PySpark job, it uses the local conda environment for that project. This way, users can install whatever libraries they like using conda and pip package managers, and then use them directly inside Spark Executors. It makes programming PySpark one step closer to the single-host experience of programming Python.
-
+Hopsworks supports cluster-wide Conda for managing Python library dependencies. Hopsworks supports the creation of projects, and each project has its own conda environment, replicated at all hosts in the cluster. When you launch a PySpark job, it uses the local conda environment for that project. This way, users can install whatever libraries they like using conda and pip package managers, and then use them directly inside Spark Executors. It makes programming PySpark one step closer to the single-host experience of programming Python.
```

Comment:
Hopsworks supports the creation of projects, ->
Hopsworks is organized around projects,

 When you launch a PySpark job, it uses the local conda environment for that project ->
 When you launch a PySpark job, both the Driver and Executors run in the conda environment for that project (the conda environment is replicated at all hosts in the cluster and available locally).